services/leecher/leecher/listener.py

```diff
@@ -28,12 +28,16 @@
 import shotgun_api3
 
 # TODO: remove hash in future since it is only used as backward compatibility
+# filters on only real events, starting with `Leeched`, not on `Recreated`
+# which have lower SG ID as they were tried, failed and should be retried
+# TODO filtering by 'leeched' or 'Leeched" doesnt work, by 'eeched' yes, wtf
 LAST_EVENT_QUERY = """query LastShotgridEvent($eventTopic: String!) {
-  events(last: 20, topics: [$eventTopic]) {
+  events(last: 3, topics: [$eventTopic], filter: "eeched") {  
```

Comment:
```suggestion
  events(last: 3, topics: [$eventTopic], filter: "Leeched") {  
```

Reply:
There is a comment with 'wtf' for this, "Leeched", "leeched" will not work as a filter value, it will return empty list, even if there are events there.